UNRELEASED.md

```diff
@@ -6,6 +6,7 @@
 
 - Add `external` prop to `ResourceList` [#2408](https://github.com/Shopify/polaris-react/pull/2408)
 - Add `onMouseEnter` and `onTouchStart` props to `Button` ([#2409](https://github.com/Shopify/polaris-react/pull/2409))
+- Improved contrast of `MessageIndicator` with a border and converted from `color` Sass mixin to CSS variables ([#2428](https://github.com/Shopify/polaris-react/pull/2428))
```

Comment:
```suggestion
- Improved contrast of `MessageIndicator` with a border ([#2428](https://github.com/Shopify/polaris-react/pull/2428))
```

We’ve been omitting changelog entries for the color system as it should be invisible to consumer unless the feature flag is present and we are considering the API unstable.